app/routes/members-area.my-account/route.module.scss

```diff
@@ -1,4 +1,82 @@
-.underConstructionMessage {
-    text-align: center;
-    font: var(--heading5);
+.smallButton {
+    padding: 5px 16px;
+}
+
+.header {
+    display: flex;
+    justify-content: space-between;
+
+    h2 {
+        font-size: 28px;
```

Comment:
it's 26px in the My Orders page.. 
wix template has 26 on one page and 28 on another 🤷